Report/Template.Rmd

```diff
@@ -243,17 +251,267 @@ datasets = read.and.clean(dataset = "easySHARE_rel6_0_0.rda", wav = 99)
 ```
 
 \section{Multidisciplinary and crossnational summary statistics}
-JULIAN
-\subsection{Theory and Design}
-\subsection{Implementation}
-\subsection{Empirical Results}
+
+## Basics
+
+We use various variables on objective and subjective health related variables. The effects of older people generally having worse health conditions than younger one remain uncompensated. Sample size and variance differ among countries. Data may be affected by different national pension schemes, that incentivize workers to extend or restrict working depending on age. 
```

Comment:
Why is it off topic? Here I am commenting on the adequacy: If a country has rather older people, the reported health is expected to be relatively worse than a country having a younger sample. Additionally, effects on labor market participation will definitely be caused by different pension schemes. This will be reflected by our data.
However, the overall design is simple: subsetting and applying functions. That is explained in the implementation part

Reply:
What do you mean with "The effects of older people generally having worse health conditions than younger one remain uncompensated"? Uncompensated by what or compared to what? Why is "data" affected by pension schemes? Do you mean participation rates? I sounds more like you are interpreting already.

Reply:
Yes I mean the participation rates are affected by the quality of the pension scheme that each worker can expect. The better the pension, the less incentive is there to work then you're old. 
With 'uncompensation' I mean that some countries may appear to have generally worse health conditions than other ones, but this is due to being older.

Reply:
Should probably refer to Kalwij and Vermeulen here, too. They said sth on the effects of pensions in one of their papers